src/components/HorizontalCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+
+import styles from './HorizontalCard.module.scss'
+
+import Button from '../Button'
+import Sticker from '../Sticker'
+
+export interface HorizontalCardProps {
+  gameImage?: React.JSX.Element
+  imageUrl?: string
+  badge?: React.ReactNode
+  button?: React.ReactNode
+  title: string
+  orderNumber?: string
+  onButtonClick?: React.MouseEventHandler<HTMLButtonElement>
+  onCardClick: React.MouseEventHandler<HTMLDivElement>
+  tone?: 'brand' | 'neutral'
+  size?: 'large' | 'small'
+}
+
+const HorizontalCard: React.FC<HorizontalCardProps> = ({
+  gameImage,
+  title,
+  badge,
+  button,
+  orderNumber,
+  onButtonClick,
+  onCardClick,
+  tone = 'neutral',
+  size = 'large'
+}) => {
+  const image = gameImage || (
+    <img src="/MoonBlastersCover.png" alt="Game Image" />
+  )
+
+  return (
+    <>
+      <div
+        className={[styles.horizontalCard, styles[tone], styles[size]].join(
+          ' '
+        )}
```

Comment:
we have a `classNames` package which is very useful for situations like this

```
import classNames from 'classNames'
```

```suggestion
        className={classNames(styles.horizontalCard, styles[tone], styles[size])}
```